gradle-enterprise-conventions-maven-extension/extension/.mvn/gradle-enterprise.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+
+<gradleEnterprise
+    xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
+  <server>
```

Comment:
Similarly, if we could dogfood the published conventions, then we wouldn't need to duplicate all of the GE configuration for this project.
This should result in a simpler publishing process overall, although it's not clear to me if/how this would create chicken-and-egg challenges in the publishing process.

Reply:
I would propose we:
- Rename the packages/groupId to `com.gradle`
- Change all `ge.example.com` to `https://ge.solutions-team.gradle.com`
- Make this releasable like any of our other plugins/extensions (`changes.md`, release jobs, release processes)
- Merge the PR with the GE configuration duplication in place
- Release version 1.0
- Follow up with a PR to consume version 1.0 of this extension from Maven Central

From there, it would behave much like any of the other plugins/extensions we use that dogfood themselves. I would likely update the README to be a little more explicit not to use the released artifact in the usage instructions.

Reply:
I think @etiennestuder is hesitant to have this sample dogfood itself. I'll let him comment here, but I think he is planning to review later today (before we spend too much time considering this path).